src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <PropertyGroup>
```

Comment:
@weshaggard and @ericstj -- curiously, with that line omitted, I had OuterLoops fail consistently on 2 different machines due to having built the product with $(TargetsWindows) false.  This was the command where they failed:
```
  build /p:WithCategories=OuterLoop
```
Yet on @StephenBonikowsky 's machine they passed.  Both of us are using the same freshly synced and cleaned master branch. By adding just this back to S.P.SM.csproj, it passed for me:
```
<PropertyGroup>  
   <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>  
</PropertyGroup>  
```
Without that line, the only way I could get OuterLoops to succeed was with this command:
```
build /p:Configuration=Windows_NT_Debug /p:OSGroup=Windows_NT /p:WithCategories=OuterLoop
```
We will investigate but it feels like it could be a race condition that gives different results on different machines. It appeared it had built both AnyOS as well as Windows_NT binaries. Neither of my machines where this failed had any unusual Environment variables set.